ngsild-payloads/equipment.jsonld

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
general comments:
- sub-attributes of an attribute must be declared like regular properties (with a `type`, a `value` if it is a property) (NB: the concise representation allows a lighter representation, but it is not yet implemented in Stellio)
- a property must have a `value` (either you'll get a 400)